config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -206,7 +206,7 @@ periodics:
       image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
       imagePullPolicy: Always
 
-- cron: '1 13 * * 2,4,6' # Run at 5:01PST (13:01 UTC) on even days
+- cron: '1 13 * * 6' # Run at 5:01PST on Fri (13:01 UTC on Sat)
```

Comment:
Again, this comment should say '5:01PST on Sat'

Reply:
fixed